symphony-bdk-legacy/symphony-api-client-java/build.gradle

```diff
@@ -0,0 +1,52 @@
+description = '[legacy] Symphony API Client'
+
+test {
+    useJUnit()
+}
+
+jacocoTestCoverageVerification {
+    violationRules {
+        rule {
+            limit {
+                counter = 'LINE'
+                value = 'COVEREDRATIO'
+                minimum = 0.42
+            }
+            element = 'CLASS'
```

Comment:
`BUNDLE` is not supported here?

Reply:
I thought it was not working as expected and not failing the build as it should but i checked again and fixed the configuration